src/Leaderboard.py

```diff
@@ -24,7 +24,7 @@ def startMatch(blueTeam: List[BallChaser], orangeTeam: List[BallChaser]) -> None
     activeMatches.insert(newActiveMatch)
 
 
-def brokenQueue(player: Member) -> str:
+def brokenQueue(player: Union[Member, str]) -> str:
```

Comment:
If we can use the mentions in the mentionBrokenQueue function then keeping this to use just Member would be better.

Reply:
This is related to my comment above.